### PR TITLE
add readme with main transition information

### PR DESCRIPTION
**Pull request by eteq**
This adds a readme with a bit of information about the repo and particularly a note for anyone who cloned before the switch to `main` (which I will do at the same time as merging this).

I'll push a similar change to `jdat_notebooks`, but we can review it here.

### CODE REVIEW

**Review by robelgeda — APPROVED** [review]
This looks good to me, Thank you!

File: README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+This repository is a sandbox and drafting space for assorted efforts for STScI's data analysis tools. (Particularly, but not exclusively, for the Data Analysis Tools Branch.)
+
+## If you locally cloned this repo before 5 Feb 2021
+
+The primary branch for this repo has been transitioned from ``master`` to ``main``.  If you have a local clone of this repository and want to keep your local branch in sync with this repo, you'll need to do the following:
+```
+% git branch -m master main
+% git fetch origin
+% git branch -u origin/main main
+```
```
Comment (robelgeda) [inline README.md:12]: A minor comment is that the "%" might confuse some people. Believe it or not, I have gotten help desk questions related to "bash$" in the instructions.

Comment (eteq) [reply README.md:12]: Oh good point.  Maybe I'll just remove that and add something like "in your terminal".

**Comment by eteq** [comment]
Thanks for the review, @robelgeda !  Merging now and then I'll make the change of the branch names (both here and jdat_notebooks)

**Pull request merged**